src/OpenAL/Extensions/OpenTK.OpenAL.Extensions.Enumeration/Enumeration.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using AdvancedDLSupport;
+using OpenTK.OpenAL.Native;
+using OpenTK.OpenAL.Native.Extensions;
+
+namespace OpenTK.OpenAL.Extensions.Enumeration
+{
+    /// <summary>
+    /// Exposes the API in the Enumeration extension.
+    /// </summary>
+    public abstract class Enumeration : ExtensionBase, IEnumerationContextState
+    {
+        /// <inheritdoc cref="ExtensionBase"/>
+        protected Enumeration(string path, ImplementationOptions options) : base(path, options)
+        {
+        }
+
+        /// <inheritdoc />
+        public abstract unsafe char* GetString(void* device, GetEnumerationContextString param);
```

Comment:
here I think a name like GetStringPointer would be more suitable(and use an explicit entrypoint name), because from C# perspective it is definitely not a string.
Or/And we could use the possibility to use the option to marshal string return values, right?

Reply:
Yeah, using the string return is probably a safer bet.